dts/arm/nordic/nrf54l_common.dtsi

```diff
@@ -18,7 +18,7 @@
 			rram0: rram@0 {
 				compatible = "soc-nv-flash";
 				erase-block-size = <4096>;
-				write-block-size = <1>;
+				write-block-size = <16>;
```

Comment:
This device **can** write in singular bytes right? So why is this being changed so it needs to write in 16 byte multiples?

Reply:
The device can, but driver shouldn't - as native wbs for RRAMC is 16B.